src/cmds.c

```diff
@@ -744,9 +744,11 @@ static void cmd_console(struct userrec *u, int idx, char *par)
 
 static void cmd_pls_bot(struct userrec *u, int idx, char *par)
 {
-  char *handle, *addr, *port, *relay, *host;
+  char *handle, *addr, *port, *relay, *relay2, *host;
+  char saddr[sizeof(struct in_addr)];
```

Comment:
this should simply be: struct in6_addr saddr or struct in_addr saddr, depending on #ifdev IPV6

Reply:
I just need a random buffer, I don't use the struct... isn't it 'cheaper' to do it this way?